package.json

```diff
@@ -1,5 +1,8 @@
 {
   "name": "@nib-styles/header-small",
   "version": "0.0.1",
```

Comment:
Sweet! We should definitely inline it with the component then... Feel free to leave this for a later date tho sonny if its impacting story delivery.